File: lista1_js/26.dias.js
//26. Leia um número inteiro de dias, calcule e escreva quantas semanas e quantos dias ele corresponde.

import { question } from "readline-sync"

//Entrada
const dias = Number(question('Informe a quantidade de dias: '))


//Processamento
const semanas = Math.floor(dias / 7)
const d = dias % 7

//Saída
console.log(`\n>>> O equivalente a essa quantidade de dias é de ${semanas.toFixed(0)} semana(s) e ${d.toFixed(0)} dia(s).`)

File: lista1_py/26.dias.py
#26. Leia um número inteiro de dias, calcule e escreva quantas semanas e quantos dias ele corresponde.

#Entrada
dias = int(input('Informe a quantidade de dias: '))

#Processamento
semanas = dias // 7
d = dias % 7

#Saída
print(f'\n>>> O equivalente a essa quantidade de dias é de {semanas} semana(s) e {d} dia(s).')